packages/jest-util/src/index.ts

```diff
@@ -21,7 +21,7 @@ export {default as deepCyclicCopy} from './deepCyclicCopy';
 export {default as convertDescriptorToString} from './convertDescriptorToString';
 export {specialChars};
 export {default as replacePathSepForGlob} from './replacePathSepForGlob';
-export {default as testPathPatternToRegExp} from './testPathPatternToRegExp';
+export {default as TestPathPatterns} from './TestPathPatterns';
```

Comment:
needs to update https://github.com/jestjs/jest/blob/main/packages/jest-util/Readme.md

Reply:
042fddd47df71e4d89ed8fba2ed3b1ad463f3099